core/server/common/src/main/webapp/js/browse.js

```diff
@@ -0,0 +1,222 @@
+/**
+ *
+ rebuild the browse page navigator pill like breadcrumbs.
+ *
+ @param {string} path current navigator path
+ *
+ */
+function rebuildNavPill(path) {
```

Comment:
Address your advise, I will think about to convert this to `React` in the next PR. So let it be it.